ui/imports/shared/views/chat/MessageView.qml

```diff
@@ -13,6 +13,7 @@ import shared.controls.chat 1.0
 Column {
     id: root
     width: parent.width
+    height: visible ? childrenRect.height: 0
```

Comment:
@Khushboo-dev-cpp I'm sorry about my... old-knowledges-mention :)
But anyway, i still see a problems:
```
    Connections {
        enabled: !!root.rootStore
        target: !!root.rootStore ? root.chatsModel.messageView : null
        onHideMessage: {
            // This hack is used because message_list deleteMessage sometimes does not remove the messages (there might be an issue with the delegate model)
            if(mId === messageId){
                root.visible = 0;
                root.height = 0;
            }
        }
    }
``` here your binding on visible will be destroyed

Reply:
the best way i think it's set to `Loader` in this file `active: root.visible`

Reply:
Hey in the code snipper you showed above, happen only when a message is deleted , I think its added because it doesn't disappear, for these 2 situations to occur simultaneously are not really possible. But I agree that setting loader active false makes more sense.

Reply:
Its done @borismelnik :)